### PR TITLE
GUI help

### PR DESCRIPTION
**Pull request by HEdingfield**
Adds help pane to config creator GUI, fixing #98.

### CODE REVIEW

**Comment by moldover** [comment]
Louis can't be stopped.

File: src/main/java/com/rcv/GuiMainController.java [hunk 1]
```diff
@@ -41,15 +41,15 @@
 
   // text area which communicates the status of the tabulator's operations
   @FXML
-  private TextArea textStatus;
+  private TextArea textAreaStatus;
```
Comment (moldover) [inline src/main/java/com/rcv/GuiMainController.java]: Is the textAreaStatus scrollable (or otherwise expandable) or more of a single-line status?

Comment (HEdingfield) [reply src/main/java/com/rcv/GuiMainController.java]: Yep, TextAreas have that functionality out of the box with JavaFX. It's great!

**Comment by HEdingfield** [comment]
I will happily accept Louis's presumably drugged-out post-surgery approvals.

**Pull request merged**